build.sbt

```diff
@@ -14,6 +16,13 @@ val nexusUrl = Option(System.getProperty("nexusUrl"))
 //TODO: this is pretty clunky, but works so far for our case...
 val nexusHost = nexusUrl.map(_.replaceAll("http[s]?://", "").replaceAll("[:/].*", ""))
 
+//Docker release configuration
```

Comment:
all these should be really configurable?

Reply:
Yep, it's easier to run script with docker tag and version of application (automate configurable by travis) than change files with these information.